CONTRIBUTING.md

```diff
@@ -78,12 +78,12 @@ The name of branch should be semantic, avoiding words like 'update' or 'tmp'. We
 
 ### 5. Make your changes
 
-Now you can create your patch, including appropriate test cases in the new branch. Please read and follow our [Code Rules](#rules).
+Now you can code. Please read and follow our [Code Rules](#rules).
 
-After you finish your modification, run the test and ensure that all tests pass.
+After you finish your modification, please prettify the code.
 
 ```shell
-npm run test
+yarn run prettier
```

Comment:
Since we don't have any tests yet, "npm run test" step is not supposed to be here. 

Substituting it with "yarn run prettier" is not a bad idea...right?

Reply:
Yes, you're right.